lib/realtime/server.ex

```diff
@@ -214,7 +235,17 @@ defmodule Realtime.Server do
   def lookup({table, {:vehicle, vehicle_or_ghost_id}}) do
     {table, :logged_in_vehicles}
     |> lookup()
-    |> Enum.filter(&(&1.id == vehicle_or_ghost_id))
+    |> Enum.find(&(&1.id == vehicle_or_ghost_id))
+    |> List.wrap()
```

Comment:
praise: I like the switch to `find` over `filter`!
question (non-blocking): Is the `List.wrap()` call still necessary?

Reply:
Yes, since these are still expected to return a list I basically use `List.wrap/1` just to wrap the singleton value in a list, or turn `nil` into an empty list.